Dockerfile

```diff
@@ -102,6 +102,8 @@ RUN /etc/scripts/import_certificates.sh \
         /etc/scripts/import_certificates.sh /tmp/certificates/; \
     fi
 
+ENTRYPOINT [ "/bin/bash" ]
```

Comment:
An explanation in the commit message *why* to have an entrypoint in the base layer would have been nice.